linetools/isgm/io.py

```diff
@@ -41,3 +51,83 @@ def abssys_from_json(filename):
 
     # Return
     return abs_sys
+
+
+def read_joebvp(filename, coord, llist=None, specfile=None, chk_vel=False):
```

Comment:
Name should be something like: `components_from_joebvp()` (Note that we already have `isgm.utils.joebvp_from_components()`. Maybe worth using both in the test case.

Reply:
renamed a few things as suggested